llvm/lib/ProfileData/InstrProfWriter.cpp

```diff
@@ -414,6 +414,145 @@ static void setSummary(IndexedInstrProf::Summary *TheSummary,
     TheSummary->setEntry(I, Res[I]);
 }
 
+namespace {
```

Comment:
The LLVM style guide prefers static functions over anonymous namespaces. 
https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

[My comment on a previous PR was not accurate, sorry for the confusion]

Reply:
Thank you for pointing this out.  Fixed in the latest iteration.